python/hsfs/core/feature_descriptive_statistics.py

```diff
@@ -25,116 +25,97 @@ class FeatureDescriptiveStatistics:
     def __init__(
         self,
         feature_type: str,
-        min: float,
-        max: float,
-        sum: float,
         count: int,
-        mean: float,
-        stddev: float,
+        end_time: Union[int, datetime, date, str],
```

Comment:
Is it not used for filtering?

Reply:
Yes, but `end_time` is not optional, so it has to be placed before the optional parameters.